service/storage_service.cc

```diff
@@ -2007,18 +2007,28 @@ future<> storage_service::bootstrap(std::unordered_set<token>& bootstrap_tokens,
 }
 
 future<std::unordered_map<dht::token_range, inet_address_vector_replica_set>>
-storage_service::get_range_to_address_map(const sstring& keyspace) const {
-    return get_range_to_address_map(_db.local().find_keyspace(keyspace).get_vnode_effective_replication_map());
+storage_service::get_range_to_address_map(const seastar::sstring& keyspace, const seastar::sstring& table) const {
```

Comment:
I think that the only caller to this function is no api/storage_service.cc
So it would be simpler to just lookup the keyspace/table and get the effective_replication_map in the api layer
and call `get_range_to_address_map` instead with the right e_r_m, since we already know which path to choose there after parsing the args.

Reply:
Sure, I will do that :+1: